LEGO1/mxregionlist.h

```diff
@@ -23,13 +23,17 @@ class MxRegionList : public MxPtrList<MxRegionTopBottom> {
 	static void Destroy(MxRegionTopBottom*);
 };
 
-// VTABLE: LEGO1 0x100dcb88
-// class MxListCursorChildChild<MxRegionTopBottom *>
-typedef MxListCursorChildChild<MxRegionTopBottom*> MxRegionListCursor;
+// VTABLE: LEGO1 0x100dcb70
+// class MxPtrListCursor<MxRegionTopBottom>
 
-// VTABLE: LEGO1 0x100dcc10
-// class MxListCursorChildChild<MxRegionLeftRight *>
-typedef MxListCursorChildChild<MxRegionLeftRight*> MxRegionLeftRightListCursor;
+// VTABLE: LEGO1 0x100dcba0
+// class MxListCursor<MxRegionTopBottom *>
+
+// VTABLE: LEGO1 0x100dcb88
+class MxRegionListCursor : public MxPtrListCursor<MxRegionTopBottom> {
+public:
+	MxRegionListCursor(MxRegionList* p_list) : MxPtrListCursor<MxRegionTopBottom>(p_list){};
```

Comment:
Apparently if you change the type from `MxRegionList*` to `MxPtrList<MxRegionTopBottom>*` the match of `vtable18` bounces back to 80%+. I have no earthly idea why that is, but I think I'd make the change and add a comment explaining that using `MxRegionList*` somehow leads to a significantly worse match. Maybe we can figure it out later.

Reply:
This is done. Only a minor drop in accuracy instead of a plunge. It might be having trouble on the cast to/from `MxRegionList` in the initializer. I left the decomp markers as they were; we'll be back in this file later to finish it up anyway.

Reply:
I think you don't need the dummy class. The code you had before worked for me, I literally only changed the type the constructor took. Can you reproduce that?